queue.c

```diff
@@ -2728,28 +2728,30 @@ BaseType_t xQueueIsQueueFullFromISR( const QueueHandle_t xQueue )
         UBaseType_t ux;
 
         configASSERT( xQueue );
-        configASSERT( pcQueueName );
 
         QueueRegistryItem_t * pxEntryToWrite = NULL;
 
-        /* See if there is an empty space in the registry.  A NULL name denotes
-         * a free slot. */
-        for( ux = ( UBaseType_t ) 0U; ux < ( UBaseType_t ) configQUEUE_REGISTRY_SIZE; ux++ )
+        if( NULL != pcQueueName )
         {
-            /* Replace an existing entry if the queue is already in the registry. */
-            if( xQueueRegistry[ ux ].xHandle == xQueue )
-            {
-                pxEntryToWrite = &( xQueueRegistry[ ux ] );
-                break;
-            }
-            /* Otherwise, store in the next empty location */
-            else if( ( NULL == pxEntryToWrite ) && ( xQueueRegistry[ ux ].pcQueueName == NULL ) )
+            /* See if there is an empty space in the registry.  A NULL name denotes
+             * a free slot. */
+            for( ux = ( UBaseType_t ) 0U; ux < ( UBaseType_t ) configQUEUE_REGISTRY_SIZE; ux++ )
             {
-                pxEntryToWrite = &( xQueueRegistry[ ux ] );
-            }
-            else
-            {
-                mtCOVERAGE_TEST_MARKER();
+                /* Replace an existing entry if the queue is already in the registry. */
+                if( xQueueRegistry[ ux ].xHandle == xQueue )
+                {
+                    pxEntryToWrite = &( xQueueRegistry[ ux ] );
+                    break;
+                }
+                /* Otherwise, store in the next empty location */
+                else if( ( NULL == pxEntryToWrite ) && ( xQueueRegistry[ ux ].pcQueueName == NULL ) )

```

Comment:
Please make this line consistent with itself - one side of the && has the NULL on the left side, while the other has the NULL on the right side.  For better or worse the kernel has always used the latter (also relevant to line 2743) although I understand the wisdom of doing it the other way around.

Reply:
Done.